packages/fiori/src/themes/WizardTab.css

```diff
@@ -126,6 +128,15 @@
 
 .ui5-wiz-step-title-text {
 	font-family: var(--_ui5_wiz_tab_title_font_family);
+	white-space: normal;
+    display: -webkit-box;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
+    overflow: hidden;
+}
+
+.ui5-wiz-step-title-text-optional {
```

Comment:
this selector should become something like:

`:host([subtitle-text]) .ui5-wiz-step-title-text {} `